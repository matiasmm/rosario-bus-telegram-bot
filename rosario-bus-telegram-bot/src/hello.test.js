import hello from './hello';

describe('hello', () => {
  it('executes as expected', async () => {
    const response = await hello({});
    expect(response).toMatchSnapshot();
  });
});
