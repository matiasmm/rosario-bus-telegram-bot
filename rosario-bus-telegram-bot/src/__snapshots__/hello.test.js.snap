// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hello executes as expected 1`] = `
Object {
  "body": "{\\"message\\":\\"Go Serverless! Your function executed successfully!\\",\\"input\\":{}}",
  "statusCode": 200,
}
`;
